docs/configuration.rst

```diff
@@ -44,6 +45,10 @@ Additionally, you can run an executable function at startup of rope.
         # Do whatever you like here!
 
 
+pytool.toml 
+-----------
+If neither a config.py or a pyproject.toml is present, rope will use a pytool.toml in the users configuration directory.
```

Comment:
Where is the "user's configuration directory"? I think we should document the default location, or how the user can get the location here, for example something like:

    If neither a config.py or a pyproject.toml is present, rope will use a pytool.toml in the user's configuration directory (i.e. `${XDG_CONFIG_HOME:-~/.config}/pytool.toml`).